main.js

```diff
@@ -1,8 +1,11 @@
-const oOverlay = require('./src/js/overlay');
+
+import Overlay from './src/js/overlay';
+
 const constructAll = function() {
-	oOverlay.init();
+	Overlay.init();
 	document.removeEventListener('o.DOMContentLoaded', constructAll);
 };
 
 document.addEventListener('o.DOMContentLoaded', constructAll);
-module.exports = oOverlay;
+
+export default Overlay;
```

Comment:
I think we can remove the import from above and replace this export with:
That way the reader of the code can see that we are not modifying the imported/exported `Overlay` at all.
```suggestion
export { default } from './src/js/overlay';
```

Reply:
Oh, we can't remove the import because we use the `init` method from `Overlay`.

Reply:
Yep :) I was copying what we're doing in all the other components